packages/components/src/components/Cell/Cell.tsx

```diff
@@ -138,10 +152,10 @@ export const Cell = forwardRef<View, React.PropsWithChildren<CellProps>>(
                     ))
                 }
             >
-                {CellContent()}
+                <Cell />
```

Comment:
There is a functional difference between referring to sub-components with the jsx-style syntax (`<Something/>`) vs calling it as a method `{Something()}`. If you use jsx-style, the component will be treated by react as a separate component in the rendering tree, and you should make sure that the sub-component uses __no__ state from the parent. Calling it as a method is faster, but if the sub-component becomes large, it is better to move it completely out as a separate component instead.

The rule of thumb to use is:
If you want to treat the component as a standalone react component:
- Use JSX-style syntax
- Move the component completely outside of the parent and pass all required data as props

If your sub-component is small and you want to keep it inside the parent:
- Use method syntax
- Freely use variables defined in either the sub-component or in the parent